sailthru_content/services/sailthru_translation_service.py

```diff
@@ -58,11 +60,23 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             for seat in course_run['seats']:
                 sailthru_content_vars['price_{}'.format(seat['type'])] = seat['price']
                 sailthru_content_vars['currency_{}'.format(seat['type'])] = seat['currency']
+                if not sailthru_content_vars.get('course_type', None):
+                    sailthru_content_vars['course_type'] = seat['type']
+                else:
+                    if sailthru_content_vars['course_type'] == 'audit':
+                        sailthru_content_vars['course_type'] = seat['type']
+                    elif sailthru_content_vars['course_type'] == 'verified':
+                        sailthru_content_vars['course_type'] = seat['type']
+                    elif sailthru_content_vars['course_type'] == 'professional':
+                        sailthru_content_vars['course_type'] = seat['type']
                 # add upgrade deadline if there is one
                 if seat.get('upgrade_deadline'):
                     deadline_key = 'upgrade_deadline_{}'.format(seat['type'])
                     sailthru_content_vars[deadline_key] = self._convert_date(seat, 'upgrade_deadline')
 
+        if course_run['key'] in GFA_COURSE_LIST:
```

Comment:
What you have in the `GFA_COURSE_LIST` are not course run keys. They are part of the marketing site course about page URL. An example GFA content key is `course-v1:ASUx+ENG102x+2167B`